### ISSUE TITLE
[feature request] activate thousands separators / change dezimal separator

### ISSUE THREAD

**Issue by flonair** [kalkih/mini-graph-card#343]
I created a status with thousands separators (dots) because large numbers are easier to read that way. But these are recognized by the mini-graph as decimal separators. 

My suggestion for a feature would be automatic thousands separators:
`123,456.78
`
Optionally, it would also be cool if you could choose between dot and comma:
`123.456,78`

**Comment by Misiu** [comment]
@flonair or space. Some of us use space as a thousand separator. So this should be an option, maybe with `` (empty string) as the default.

**Comment by haty1** [comment]
I also vote for this. My preference is for a , as the seperator but could back this user option

**Comment by flonair** [comment]
> @flonair or space. Some of us use space as a thousand separator. So this should be an option, maybe with `` (empty string) as the default.

Yeah, even better. This option could be the best for all regions.

**Comment by kalkih** [comment]
Could possibly utilize [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/format), with the locale as a user option

**Comment by sjikkervisje** [comment]
Please have the card automatically follow the locale settings like the normal sensor card does. I would like it to show a dot for the thousand separator and a comma for the decimal separator which is my locale setting. Or the option to set this in the YAML>

**Comment by RomRider** [comment]
This is available in the latest beta already

**Comment by fmonday** [comment]
Seconding the thousand separator (either by following user settings or better yet as an option in the card). If it was in the beta it apparently hasn't made it into the HACS version (currently at v 0.10.0).

**Comment by fmonday** [comment]
Installed 0.11.0-dev.4 (after fighting with HACS for a while) and it works beautifully. Thanks @sjikkervisje!
